source/cli.civet

```diff
@@ -387,4 +420,18 @@ function cli
           console.error error
           process.exit 1
 
+  process.exitCode = errors
```

Comment:
Problably want to add `Math.min(errors, 255)`. I think most shells wrap at 256.

Reply:
Bash and (most?) other POSIX shells do, but notably PowerShell doesn't -- it uses 32-bit exit codes. Microsoft likes to put useful information in the high bytes of the exit code which can be annoying when invoking a .exe file from bash.